src/gr_rtl_tuner.cpp

```diff
@@ -315,6 +315,13 @@ void create_fm_device(rtl_ctx &context)
 
 void rtl_add_audio_sink(rtl_ctx_t* this_tuner, const char* device, int sampling_rate) {
     gr::audio::sink::sptr audsink = gr::audio::sink::make(sampling_rate, device);
```

Comment:
This line can throw an exception.  Maybe initialize the pointer to nullptr here instead?

Reply:
This was a typo the audsink needed to be set to null.